### PR TITLE
Add Near filter using geo points for filter

### PR DESCRIPTION
**Pull request by vnegrisolo**
- [x] new `Near` filter for filtering using geo point

reference: https://github.com/LoveMondays/love-mondays/issues/3436

### CODE REVIEW

File: lib/elasticband/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Elasticband
-  VERSION = '0.2.0'
+  VERSION = '0.2.1'
```
Comment (rubensmabueno) [inline lib/elasticband/version.rb]: Isn't this a new feature?

Comment (vnegrisolo) [inline lib/elasticband/version.rb]: done

**Comment by rubensmabueno** [comment]
:+1:

**Comment by brennovich** [comment]
:+1:

**Pull request merged**